src/app-bar.jsx

```diff
@@ -162,8 +163,8 @@ const AppBar = React.createClass({
       // If the title is a string, wrap in an h1 tag.
       // If not, just use it as a node.
       titleElement = typeof title === 'string' || title instanceof String ?
-        <h1 style={this.prepareStyles(styles.title, styles.mainElement)}>{title}</h1> :
-        <div style={this.prepareStyles(styles.mainElement)}>{title}</div>;
+        <h1 onClick={this._onTitleTouchTap} style={this.prepareStyles(styles.title, styles.mainElement)}>{title}</h1> :
```

Comment:
@StoneCypher before I merge this, should we change these `onClick`s to `onTouchTap`s?